examples/BlazorServerApp/BrowserFilter.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Microsoft.FeatureManagement;
+
+namespace BlazorServerApp
+{
+    [FilterAlias("Browser")]
+    public class BrowserFilter : IFeatureFilter
+    {
+        private readonly UserAgentContextProvider _userAgentContextProvider;
+
+        public BrowserFilter(UserAgentContextProvider userAgentContextProvider)
+        {
+            _userAgentContextProvider = userAgentContextProvider ?? throw new ArgumentNullException(nameof(userAgentContextProvider));
+        }
+
+        public Task<bool> EvaluateAsync(FeatureFilterEvaluationContext context)
+        {
+            string userAgentContext = _userAgentContextProvider.Context;
+
+            return Task.FromResult(IsEdgeBrowser(userAgentContext));
```

Comment:
My expectation would be a filter like this takes allowed browsers as filter parameters.